code/modules/projectiles/ammo_datums.dm

```diff
@@ -2731,6 +2731,33 @@
 	shell_speed = AMMO_SPEED_TIER_2
 	added_spit_delay = 0
 
+
+/datum/ammo/xeno/acid/marking
+	name = "marking spit"
+
+	damage = 15
+	shell_speed = AMMO_SPEED_TIER_2
+	accuracy = HIT_ACCURACY_TIER_10*8
+	flags_ammo_behavior = AMMO_SKIPS_ALIENS|AMMO_IGNORE_ARMOR
+	damage_falloff = 0
+	max_range = 32
+
+/datum/ammo/xeno/acid/marking/on_hit_mob(mob/target, obj/item/projectile/impacting_projectile)
+	. = ..()
+	target.AddComponent(/datum/component/bonus_damage_stack, 100, 1 SECONDS) //subject to change
+	for(var/mob/living/carbon/xenomorph/boosting_xeno in range(5, target))
+		var/hivenumber = XENO_HIVE_NORMAL
+
+		if (!boosting_xeno.check_state())
+			break
+
+		if (boosting_xeno.hivenumber != hivenumber || boosting_xeno.stat == DEAD)
+			continue
+
+		to_chat(boosting_xeno, SPAN_XENOBOLDNOTICE("You feel energized as the acid hits [target]!"))
+		boosting_xeno.gain_health(30)
+		target.balloon_alert(boosting_xeno, "gain energy!")
```

Comment:
This alert needs to be far better phrased. "Carapace regenerates!" or something similar. I'm uncertain if a balloon alert is even needed but I'll talk with some others.